src/main/java/uk/gov/hmcts/darts/arm/service/impl/UnstructuredToArmProcessorImpl.java

```diff
@@ -116,17 +117,18 @@ public void processUnstructuredToArm() {
 
             updateExternalObjectDirectoryStatus(armExternalObjectDirectory, armIngestionStatus);
 
-            String filename = generateFilename(armExternalObjectDirectory);
-
+            String rawFilename = generateFilename(armExternalObjectDirectory);
+            log.info("Start of ARM Push processing for EOD {} running at: {}", armExternalObjectDirectory.getId(), LocalDateTime.now());
```

Comment:
For consistency with our Timestamps in darts-api, I think it's preferable for us to use `OffsetDateTime`.
Also the log.info would give you that UTC server time for free anyway? So maybe we don't even care for it much